libs/common-system/src/test/java/org/codice/ddf/configuration/SystemBaseUrlTest.java

```diff
@@ -27,86 +27,138 @@ public void setUp() throws Exception {
     System.setProperty("org.codice.ddf.system.hostname", "localhost");
     System.setProperty("org.codice.ddf.system.httpsPort", "8993");
     System.setProperty("org.codice.ddf.system.httpPort", "8181");
+    System.setProperty("org.codice.ddf.external.protocol", "https://");
```

Comment:
❓ Same as above

Reply:
I'm going to keep these for now because they're testing the SystemBaseUrl class.